src/main/java/com/adobe/commerce/cif/api/CartApi.java

```diff
@@ -102,6 +106,8 @@ Cart deleteCart(
     @Path("/{id}/entries")
     @ApiOperation(value = "Adds a new cart entry to an existing cart.")
     @ApiResponses(value = {
+        @ApiResponse(code = 201, message = "Created.", response = Cart.class,
+                responseHeaders = @ResponseHeader(name = "Location", description = "Location of the newly created cart entry.", response = String.class)),
```

Comment:
One problem is that we don't (yet) have an endpoint to GET a specific cart entry.

Reply:
Yes but we can update and delete the resource using a unique URL, so for now, this looks like a write-only resource in that respect.

Reply:
@santha is this already followed up? I hope I am not missing the obvious but  AFAIK we don’t yet implement the Location header.